app/domain/people/membership.rb

```diff
@@ -5,24 +5,36 @@
 #  or later. See the COPYING file at the top-level directory or at
 #  https://github.com/hitobito/hitobito_sac_cas.
 
-class People::MembershipVerifier
+class People::Membership
 
   MEMBERSHIP_ROLES = [
     Group::SektionsMitglieder::Mitglied
```

Comment:
Ev. sollten wir das in `app/domain/sac_cas.rb` pflegen

Reply:
Ansich dachte ich `SacCas` verwenden wir nur als parent wenn wir bestehendes patchen ..  :thinking:,